components/commands/commandsUtils.js

```diff
@@ -42,7 +42,8 @@ const getCommandsRegex = () => {
     checkInfos: /^!checkinfos/g,
     openSource: /^!opensource$/g,
     openSourceAddRepository: /^!addrepositorio[ \d\w]*$/g,
-    openSourceRemoveRepositoryUser: /^!removerepositoriousuario[ \d\w @a-z-A-Z]*$/g
+    openSourceRemoveRepositoryUser: /^!removerepositoriousuario[ \d\w @a-z-A-Z]*$/g,
```

Comment:
`removerepositoriousuario` why this in Portuguese?

Reply:
I still dont know too, but I have a proposal to refactor these commands, and maybe in another pull request this will be changed